Mathlib/RingTheory/UniqueFactorizationDomain.lean

```diff
@@ -1020,6 +1024,24 @@ theorem count_normalizedFactors_eq' {p x : R} (hp : p = 0 ∨ Irreducible p) (hn
   · exact count_normalizedFactors_eq hp hnorm hle hlt
 #align unique_factorization_monoid.count_normalized_factors_eq' UniqueFactorizationMonoid.count_normalizedFactors_eq'
 
+
+theorem max_powerFactor {a₀ : R} {x : R} (h : a₀ ≠ 0) (hx : Irreducible x) :
```

Comment:
I don't know if this name max_powerFactor is correct from the mathlib4 point of view.